hack/generator/azure-arm.yaml

```diff
@@ -15,12 +15,6 @@ typeFilters:
     group: microsoft.servicebus
     version: v*api20210101preview
     because: We want to export this particular preview version
-  # TODO: This is temporarily included because the SB RP doesn't support queue/topic
-  # TODO: in 20210101preview right now.
-  - action: include
-    group: microsoft.servicebus
-    version: v*api20180101preview
-    because: We want to export this particular preview version
```

Comment:
Should this change be in this PR?

Reply:
Yeah this is the old version that isn't actually needed anymore (we don't export any types from it)